vecnet/vecnet_linux_arm.go

```diff
@@ -0,0 +1,109 @@
+// Copyright 2018 The gVisor Authors.
```

Comment:
how are these different on arm,arm64? i don't see the diff from x86, but it's also a bit needle-in-haystack

Reply:
Specifically it's the syscall iovec's length size.
>Len:  uint32(l)
Len:  uint64(l)
msg.Iovlen = uint32(len(iovecs))
msg.Iovlen = uint64(len(iovecs))

There may be a better way to reduce this though, this was some quick copy paste hackery.

Reply:
#15 has examples of how to make this easier